sdk/java-sdk/src/main/java/kalix/javasdk/package-info.java

```diff
@@ -17,25 +17,25 @@
  *   <tbody>
  *     <tr class="altColor">
  *       <th class="colFirst" scope="row">Action</th>
- *       <td class="colLast"><a href="{@docRoot}/com/akkaserverless/javasdk/action/package-summary.html"><code>com.akkaserverless.javasdk.action</code></a></td>
+ *       <td class="colLast"><a href="{@docRoot}/com/akkaserverless/javasdk/action/package-summary.html"><code>kalix.javasdk.action</code></a></td>
```

Comment:
These links will have to be fixed. But I wasn't going for completeness here, yet.